test/blocks/merch/mocks/body.html

```diff
@@ -56,6 +56,11 @@ <h2>Promo prices</h2>
     href="/tools/ost?osi=0RwUma15VBOveHgpEz6cKCxATwVkUO8WO3QkI97f4Sg&type=price&promo=nicopromo">Price -
     632B3ADD940A7FBB7864AA5AD19B8D28 - All Apps </a>
 </p>
+<h2>CTAs</h2>
+<p>CTA with config default: <a class="merch cta config"
+  href="/tools/ost?osi=0RwUma15VBOveHgpEz6cKCxATwVkUO8WO3QkI97f4Sg&type=checkoutUrl">CTA Buy Now</a></p>
+<p>CTA with link-level overrides: <a class="merch cta link-overrides"
+  href="/tools/ost?osi=0RwUma15VBOveHgpEz6cKCxATwVkUO8WO3QkI97f4Sg&type=checkoutUrl&checkoutType=UCv2&workflowStep=checkout_email&marketSegment=EDU">CTA Buy Now</a></p>
```

Comment:
in `checkout_email` isn't `checkout` part implicit with `ucv2` ?  Are there other values?
if not, I recommend to keep only the step value, e.g: `email`.
Also, if we dont' keep the `checkoutType` in the OST url but as a context value, switch from `ucv2` to `ucv3` would be seamless.

Reply:
I agree, the ost link should contain only exceptional values.
the checkout type and step could indeed be a context value.